tests/Support/SupportStrTest.php

```diff
@@ -1196,8 +1197,13 @@ public function testInlineMarkdown()
 
     public function testRepeat()
     {
+        $this->assertSame('', Str::repeat('Hello', 0));
+        $this->assertSame('Hello', Str::repeat('Hello', 1));
         $this->assertSame('aaaaa', Str::repeat('a', 5));
         $this->assertSame('', Str::repeat('', 5));
+
+        $this->expectException(ValueError::class);
```

Comment:
When using `expectException()`, you should split it into another test method.

Reply:
Based on your guidance, I have done the same. Thank you for the guidance.

@crynobone